projects/freedesktop.org/pkg-config/package.yml

```diff
@@ -31,6 +31,8 @@ build:
       - --with-internal-glib
       # otherwise the defaults are based on our {{prefix}}
       - --with-pc-path=/usr/lib/pkgconfig:/usr/share/pkgconfig
+    # llvm15 changed this to hard error
+    CFLAGS: -Wno-error=int-conversion
```

Comment:
We need to version restrict this so if they release `0.29.3` with the fix we stop doing this.

We have no easy way to do that yet tho.

Reply:
The "good" news is that this should be impossible to break working code; it can only allow broken code to compile.